reactor-core/src/main/java/reactor/core/publisher/FluxFlattenIterable.java

```diff
@@ -658,6 +697,7 @@ public boolean isEmpty() {
 		@Nullable
 		public R poll() {
 			Iterator<? extends R> it = current;
+			boolean itFinite = currentKnownToBeFinite;
```

Comment:
Isn't this initialization redundant? It will be updated before its used.

Reply:
indeed, fixed